### PR TITLE
Add Tunnels

### PR DESCRIPTION
**Pull request by parro-it**
An app to manage a list of ssh tunnel to your remote machines and open them all at once.

https://github.com/parro-it/tunnels

### CODE REVIEW

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/awesome-electron/blob/master/contributing.md

> If you just created something, wait at least a couple of weeks before submitting.

> Submitted open source apps should have a English readme, screenshot of the app in the readme, and a binary for at least one OS, preferably OS X, Linux and Windows.

The UI could use some work. I would strongly recommend using an UI kit like http://photonkit.com.

-

Ping me here in some weeks and I'll reopen.

File: readme.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ Some good apps written with Electron.
 - [Eintopf](https://github.com/mazehall/eintopf) - Manage development projects with Docker.
 - [WhatsDesktop](https://github.com/mawie81/whatsdesktop) - Unofficial WhatsApp app.
 - [Sqlectron](https://github.com/sqlectron/sqlectron-gui) - SQL client.
+- [Tunnels](https://github.com/parro-it/tunnels) - Manage your ssh tunnels.
```
Comment (sindresorhus) [inline readme.md]: > Manage SSH tunnels.

**Comment by parro-it** [comment]
I will, and I don't want to annoy you, however I'm already using http://photonkit.com, 
and I started to develop the app some months ago... Could I ask you if you think the app could be useful?

**Comment by sindresorhus** [comment]
> and I started to develop the app some months ago

The rule is there to ensure it wasn't just a code dump. I base it on the first release: https://github.com/parro-it/tunnels/commit/1090b7dd89efc48721293d292cdf7ef7262759a3

> Could I ask you if you think the app could be useful?

I personally wouldn't use it right now because of the current UI, but I have very high UI standards for what I use. I'm sure it would come in useful for many :)

**Comment by parro-it** [comment]
> The rule is there to ensure it wasn't just a code dump. I base it on the first release: parro-it/tunnels@1090b7d

That's fair, I just removed lot of commits from the repo, but I will wait

> I personally wouldn't use it right now because of the current UI, but I have very high UI standards for what I use. I'm sure it would come in useful for many :)

Nice, I'll try to improve on that, thank you.

**Comment by sindresorhus** [comment]
Some quick tips regarding the UI. Make sure everything is aligned. Don't use both icons and text in the buttons. Don't use green and red buttons. Blue for main action, gray for the rest. Don't use buttons in the list view, icons could be enough or just an action button to opens a context menu with more. The "Edit tunnel" window is too tall. Try to optimize the space by combining related input on the same lines. The "Check" button seem moot, just check in realtime. See the Photon example on how to place Save/Cancel buttons at the bottom of the window. In the "Tunnel list" window, place the add button at the bottom in a footer toolbar, like you did in the "Edit" window.

**Comment by parro-it** [comment]
Thank you for your suggestions and for your time! :satisfied:

**Pull request closed without merging**